### PR TITLE
ref: Use new sentry-sdk

### PR DESCRIPTION
**Pull request by untitaker**

### CODE REVIEW

File: poetry.lock [hunk 1]
```diff
@@ -79,7 +79,7 @@ python-versions = "*"
 version = "1.4.3"
 
 [[package]]
-category = "dev"
+category = "main"
```
Comment (untitaker) [inline poetry.lock]: poetry did this, I am innocent

File: zeus/config.py [hunk 2]
```diff
@@ -33,7 +31,6 @@
 mail = Mail()
 nplusone = NPlusOne()
 redis = Redis()
-sentry = Sentry(logging=True, level=logging.ERROR, wrap_wsgi=True)
```
Comment (untitaker) [inline zeus/config.py]: I am not sure if having the level here at `error` and further down at `warn` was intentional fine-tuning. It's now `error` for all loggers

File: zeus/config.py [hunk 3]
```diff
@@ -368,3 +349,32 @@ def configure_scout(app):
...
+    from sentry_sdk import init
+    from sentry_sdk.integrations.flask import FlaskIntegration
+
+    release = os.environ.get("BUILD_REVISION") or None
+    if not release:
+        try:
+            import subprocess
+
+            release = str(
+                subprocess.check_output(["git", "describe", "--always"])
```
Comment (untitaker) [inline zeus/config.py]: Idk if this is exactly equal to what raven did, but afaict this doesn't run in prod

File: poetry.lock (hunk 1 above)
Comment (dcramer) [reply poetry.lock]: 🤣

**Pull request merged**